lib/preprocessors/glimmer.js

```diff
@@ -135,7 +160,27 @@ function mapRange(messages, filename) {
     );
   }
 
-  return flattened.map((message) => {
+  const filtered = flattened.filter(
+    (it) =>
+      it.ruleId === 'no-undef' ||
```

Comment:
Can you comment to explain why `no-undef` is the only rule included here?

Reply:
i think @NullVoxPopuli should answer :)

Reply:
Is the logic here "ember-template-lint should give us errors from inside a template context"?

are we certain there will not be other eslint rules written in the future that may require us to update this logic?

Reply:
@hmajoros exactly -- the only thing template-lint can't know about is what is in scope -- so the template-lint equiv of no-undef is _disabled_ for gjs/gts

Reply:
There is a comment here already: https://github.com/ember-cli/eslint-plugin-ember/blob/208e6b6e8a017f84671891c42953913a5eb05e3f/lib/preprocessors/glimmer.js#L88

Reply:
Should i move it down to this code?

Reply:
I think two improvements would help:

1. Update the comment to explain why we need to "allow-list the rules that check for undefined identifiers".
2. Move the rule name into an array with a descriptive name:

```js
const RULES_THAT_CHECK_FOR_UNDEFINED_IDENTIFIERS = ['no-undef'];
```

This name is self-documenting and an array would be reusable and easily expandable.